src/Microsoft.Kusto.ServiceLayer/DataSource/DataSourceIntellisense/KustoIntellisenseHelper.cs

```diff
@@ -295,22 +266,27 @@ public static CompletionItemKind CreateCompletionItemKind(CompletionKind kustoKi
         /// <summary>
         /// Loads the schema for the specified database and returns a new <see cref="GlobalState"/> with the database added or updated.
         /// </summary>
```

Comment:
Formatting changes

Reply:
If we are formatting for better reading, can we do one param per line.